policyengine_us/variables/gov/states/az/tax/income/rebates/az_one_time_families_tax_rebates.py

```diff
@@ -0,0 +1,35 @@
+from policyengine_us.model_api import *
+
+
+class az_one_time_families_tax_rebates(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona one-time families tax rebates"
+    unit = USD
+    definition_period = YEAR
+    defined_for = "az_income_tax_rebates_eligibility"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.az.tax.income.rebates
+        person = tax_unit.members
+        dependent = person("is_tax_unit_dependent", period)
+        age = person("age", period) - 2  # the age at the end of 2021
```

Comment:
add comments explaining what we do here:
"Only children born after 2021 are considered for the program ..."